app/services/get-capacity-view.js

```diff
@@ -14,11 +12,7 @@ module.exports = function (id, capacityDateRange, organisationLevel) {
     throw new Error(organisationLevel + ' should be offender-manager, region, team, ldu or hmpps')
   }
 
-  if (organisationLevel === routeType.OFFENDER_MANAGER.name) {
-    workloadReportsPromise = getIndividualWorkloadReports(id, capacityDateRange.capacityFromDate.toISOString(), capacityDateRange.capacityToDate.toISOString())
-  } else {
-    workloadReportsPromise = getWorkloadReports(id, capacityDateRange.capacityFromDate.toISOString(), capacityDateRange.capacityToDate.toISOString(), organisationLevel)
-  }
+  workloadReportsPromise = getWorkloadReports(id, capacityDateRange.capacityFromDate.toISOString(), capacityDateRange.capacityToDate.toISOString(), organisationLevel)
```

Comment:
Do we not still need to have separate individual and organisation?

Reply:
I don't think so. The group query should be ok to call the individual view now that it exists. I have updated how the individual query is called (new param, 'offender-manager') and added the view name to the offender-manager constant. This got all tests passing and screens look ok